samples/bluetooth/broadcast_assistant/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2024 Demant A/S
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "Bluetooth: Broadcast Assistant"
+
+config SELECT_SOURCE_NAME
+	string "Selected Broadcast Source name"
+	default "Broadcast Audio Source"
```

Comment:
I am not sure it's a good idea to "duplicate" default values across samples. I'd suggest to look at  comments in #67581 and follow recommendation from @Thalley

Reply:
I did this because developers might sit in an environment with a lot of unwanted sinks and sources (like where I work, for example ;)).  I can change it to the empty string and adjust the documentation with a recommendation to type the ones names from the other samples, in case the user is in a similar environment.  It was done to get a nice "plug'n'play" experience - but I see your point (to keep it clean and separate).

It's a quick remove, if needed - thoughts on the above? (I'm OK either way)

Reply:
I looked at a few other samples - and I think you are right: I'll make it the empty string and then just make a note in the docs.